assets/src/data/eventespresso/schema/selectors.js

```diff
@@ -152,6 +152,24 @@ export function isRequestingRelationEndpointForEntityId(
 	);
 }
 
+/**
+ * Selector returning the relation response type for the given relation.
+ *
+ * @param {Object} state
+ * @param {string} modelName
+ * @param {string} relationName
+ * @return {string} The type for the relation returned for the given model and
+ * relation.
+ */
+export const getRelationResponseType = ( state, modelName, relationName ) => {
+	modelName = singularModelName( modelName );
+	relationName = pluralModelName( relationName );
+	const relationSchema = getRelationSchema( state, modelName, relationName );
+	return relationSchema !== null ?
+		relationSchema.type :
+		'';
```

Comment:
Why not just return `array` here as the default, instead of having to convert elsewhere?
I'm assuming it's better to know the real type and then be at liberty to change it if need be, even if that ends up being repeated in several places.
Another option might be to add a `default` parameter to this function (and any others that call it),
so that the default can be set ahead of time, and save additional processing elsewhere. 

ie:
```js
export const getRelationResponseType = ( state, modelName, relationName, default = '' ) => {
	modelName = singularModelName( modelName );
	relationName = pluralModelName( relationName );
	const relationSchema = getRelationSchema( state, modelName, relationName );
	return relationSchema !== null ?
		relationSchema.type :
		default ;
};
```

Reply:
I don't think this should have a default as its schematic information returning the expected return type.  How would client know what the server is going to return in order to set a default?  I think it's better that this function has a return value that is consistent when there is no response type available for the given arguments.  Caller code then can decide what it's going to do when this returns that value.  In this case, I'm returning an empty string so it shares the same data type it would be if there _was_ a non-empty value.